docs/getting-started.md

```diff
@@ -105,3 +110,35 @@ Or to start the auto-restarting development server:
 ```sh
 make start-dev
 ```
+
+Run Docker for development
+----------------------
+
+The `Dockerfile.dev` and `docker-compose.yml` file at the root of the project work together to start the
+development server with a MySQL database.
+
+Note: so far this has been built with a bias for ARM64/Apple Silicon machines. 
+
+Prerequisites:
+
+- [Docker Desktop](https://docs.docker.com/desktop/) or CLI
+
+### Setup
+1. Copy the `.env.defaults` file to `.env` and edit the variables to your liking. 
```

Comment:
I think we should steer people away from using `.env` in docker contexts. Docker / docker-compose has their own way to set environment variables and it's much better for docker-compose users to handle this centrally.

Maybe the metabase docker-compose .yml is a good template to go off:

https://www.metabase.com/docs/latest/installation-and-operation/running-metabase-on-docker

Reply:
oh true! yeah `.env` seems to be very much a local thing
At work, people seem to stack these for different envs, set them on the platform, or use a separate file and reference it